### PR TITLE
Better error messaging for incorrect time range input at AppLens

### PR DESCRIPTION
**Pull request by michimune**
This is a PR for the work item 1393: Better error messaging for incorrect time range input.

This is only for AppLens, not for App Service Diagnostics.

### CODE REVIEW

**Comment by stevenernst131** [comment]
This code is shared between both applens and app service diagnostics. I think we should come up with a solution that works for both. Right now this will also show up for App Service Diagnostics. 

Praveen had also asked me to remove the word 'detector' for external, since we don't call them that to customers anywhere else.

File: AngularApp/projects/diagnostic-data/src/lib/components/detector-view/detector-view.component.html [hunk 1]
```diff
@@ -45,5 +45,5 @@ <h1 *ngIf="!isSystemInvoker">
 <loader-view style="margin-top:300px" *ngIf="!detectorDataLocalCopy && !errorState" message="Loading Detector..."></loader-view>
 
 <ng-container *ngIf="!detectorDataLocalCopy && errorState">
-  Error loading detector
+  Error loading detector: <span class="critical-color">{{errorState.error}}</span>
```
Comment (stevenernst131) [inline AngularApp/projects/diagnostic-data/src/lib/components/detector-view/detector-view.component.html]: You may want to do something like this, just to make sure the error property exists:

```(html)
<span class="critical-color" *ngIf="errorState.error">{{errorState.error}}</span>
```

Comment (michimune) [reply AngularApp/projects/diagnostic-data/src/lib/components/detector-view/detector-view.component.html]: Thanks! So do we also want to change the message "Error loading detector" to remove the word 'detector'?

**Comment by michimune** [comment]
Thank you!

**Pull request merged**